src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer started working");
+    }
+
```

Comment:
please add `@Override` annotation also for the method `stopWork()`. fix in all classes that extends from `Machine`